src/index.html

```diff
@@ -2,13 +2,90 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@500&family=Roboto:wght@500&display=swap"
+          rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a 
+        href="#home"
+        class="logo"
+      >
+        <img 
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a 
+              href="#apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung"
+               class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones"
+               class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>          
+          <li class="nav__item">
+            <a href="#laptops&Computers"
+               class="nav__link"
+               data-qa="hover"
+            >
```

Comment:
```suggestion
            <a
               href="#laptops&Computers"
               class="nav__link"
               data-qa="hover"
            >
```
1 attr per line - fix it in all places